test/mock_query_test.dart

```diff
@@ -1622,6 +1622,70 @@ void main() {
     });
   });
 
+  group('aggregate queries', () {
+    test('should return null for unspecified sum field', () async {
+      final firestore = FakeFirebaseFirestore();
```

Comment:
I was going to suggest using setUpAll, but then I saw your last test case, which uses different data.

```dart
late FakeFirebaseFirestore firestore;
late CollectionReference collection;

setUpAll(() async {
  firestore = FakeFirebaseFirestore();
  collection = firestore.collection('firestore-example-app');

  await collection.add({'name': 'Banana', 'likes': 1, 'dislikes': 4});
  await collection.add({'name': 'Avocado', 'likes': 2, 'dislikes': 5});
  await collection.add({'name': 'Mango', 'likes': 3, 'dislikes': 6});
});

test('should return null for unspecified sum field', () async {
  final _sum = await collection.aggregate(sum('likes')).get();
  expect(_sum.getSum('dislikes'), isNull);
  expect(_sum.count, 3);
});
```